server/core/region.go

```diff
@@ -960,6 +960,26 @@ func (r *RegionsInfo) GetFollower(storeID uint64, region *RegionInfo) *RegionInf
 	return nil
 }
 
+// GetLoads returns loads from region
+func (r *RegionInfo) GetLoads() []float64 {
+	return []float64{
+		float64(r.GetBytesRead()),
```

Comment:
Can this unity to `GetLoadbyKind`?

Reply:
I will move `kind.go` of statistic to core in a later pr to avoid circular dependencies